submissions/PreciousAisosaErhabor/HTML/PAgent_ng.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html>
+<head>
+ 
+<title>https://PaGent.ng/registration</title>
+  <img scr="PaGent.ng.png" height="10"; width="10"/>
```

Comment:
@Epresh `scr` should be `src`. Please commit this image file `PaGent.ng.png` to github as well and push your changes. Remove the `;` after the height attribute value. Test your code on a browser to confirm that it works as expected.